python/GafferImageTest/ImageWriterTest.py

```diff
@@ -126,7 +126,7 @@ def testTiffWrite( self ) :
 				{ 'value': "float", 'metadata': { 'oiio:BitsPerSample': IECore.IntData( 32 ) } },
 			]
 
-		self.__testExtension( "tif", "tiff", options = options, metadataToIgnore = [ "tiff:RowsPerStrip" ] )
+		self.__testExtension( "tif", "tiff", options = options, metadataToIgnore = [ "tiff:RowsPerStrip", "IPTC:Creator" ] )
```

Comment:
was it failing on master too? maybe whatever oiio version we're using now doesn't translate that field anymore?